pkg/mcs/scheduling/server/config/watcher.go

```diff
@@ -215,6 +215,9 @@ func (cw *Watcher) initializeSchedulerConfigWatcher() error {
 	cw.schedulerConfigWatcher = etcdutil.NewLoopWatcher(
 		cw.ctx, &cw.wg,
 		cw.etcdClient,
+		// NOTE: schedulerConfigPathPrefix is "/pd/{cluster_id}/scheduler_config" before.
+		// Now it is "/pd/{cluster_id}/scheduler_config/". I think This has no impact.
+		// If it needs to be fixed, I can update it.
```

Comment:
These comment will be removed after review

Reply:
I think it's best to remain consistent with what it was before.

Reply:
Done

Reply:
The previous commit had issues, so I submitted a new one. https://github.com/tikv/pd/pull/9133/commits/6b837249ae6d06cdd366b9a91eee929457eaf573